plugin/config.json

```diff
@@ -58,6 +58,13 @@
       "description": "non-empty if pre-existing volumes need to be discovered ",
 	  "value": "",
       "Settable": [ "value"]
-	}]
+	},
+	{
+      "name": "TEST_PROTOCOL_VERSION",
```

Comment:
nit (since it's not for customers) Can we stick with VDVS_ as a prefix for all keys  ?"